database/migrations/2024_07_02_231614_create_products_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('products', function (Blueprint $table) {
+            $table->id();
+            $table->string("name",50);
+            $table->integer("price");
+            $table->date("date");
```

Comment:
don't need to add date column as we already have created_at and updated_at columns therer